src/Service.Host/client/src/components/salesPackages/SalesPackage.js

```diff
@@ -0,0 +1,275 @@
+                    ) : (
+                        <Title text="Sales Package Details" />
+                    )}
+                </Grid>
+                {loading || (!item && !creating()) ? (
+                    <Grid item xs={12}>
+                        <Loading />
+                    </Grid>
+                ) : (
+                    <Fragment>
+                        {errorMessage && (
+                            <Grid item xs={12}>
+                                <ErrorCard errorMessage={errorMessage} />
+                            </Grid>
+                        )}
+                        <SnackbarMessage
+                            visible={snackbarVisible}
+                            onClose={() => setSnackbarVisible(false)}
+                            message="Save Successful"
+                        />
+                        <Grid item xs={4}>
+                            <InputField
+                                fullWidth
+                                disabled={!creating()}
+                                value={salesPackage.salesPackageId}
+                                label="Sales Package Id"
+                                helperText={
+                                    !creating()
+                                        ? 'This field cannot be changed'
+                                        : 'This field is required'
+                                }
+                                maxLength={14}
+                                onChange={handleFieldChange}
+                                propertyName="salesPackageId"
+                                error={!salesPackage.salesPackageId}
+                            />
+                        </Grid>
+                        <Grid item xs={8} />
+                        <Grid item xs={8}>
+                            <InputField
+                                value={salesPackage.description}
+                                label="Description"
+                                fullWidth
+                                rows={3}
+                                onChange={handleFieldChange}
+                                propertyName="description"
+                                maxLength={2000}
+                            />
+                        </Grid>
+                        <Grid item xs={4} />
+                        {!creating() && (
+                            <Table>
+                                <TableHead>
+                                    <TableRow>
+                                        <TableCell>Type</TableCell>
+                                        <TableCell align="right">Sequence</TableCell>
+                                        <TableCell align="right">Quantity</TableCell>
+                                    </TableRow>
+                                </TableHead>
+                                <TableBody>
+                                    {item.elements &&
```

Comment:
Shouldn't this be `salesPackage.elements` as opposed to `item.elements`

Reply:
Yes, thanks!